.github/workflows/cross-chain-arbitrum.yml

```diff
@@ -0,0 +1,299 @@
+          cache: "yarn"
+          cache-dependency-path: cross-chain/arbitrum/yarn.lock
+
+      # A workaround for transitive dependencies that use the obsolete git://
+      # prefix instead of the recommended https://
+      - name: Configure git to not use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Deploy contracts
+        run: yarn deploy
+
+  contracts-deployment-testnet:
+    needs: [contracts-build-and-test]
+    if: |
+      github.event_name == 'workflow_dispatch' 
+        && github.event.inputs.environment != 'dev'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cross-chain/arbitrum
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+          cache-dependency-path: cross-chain/arbitrum/yarn.lock
+          registry-url: "https://registry.npmjs.org"
+
+      # A workaround for transitive dependencies that use the obsolete git://
+      # prefix instead of the recommended https://
+      - name: Configure git to not use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Get upstream packages versions
+        uses: keep-network/ci/actions/upstream-builds-query@v2
+        id: upstream-builds-query
+        with:
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: tbtc-v2-contracts-version = github.com/keep-network/tbtc-v2#version
+
+      - name: Resolve latest contracts
+        run: yarn upgrade @keep-network/tbtc-v2@${{ steps.upstream-builds-query.outputs.tbtc-v2-contracts-version }}
+
+      - name: Deploy contracts
+        env:
+          CHAIN_API_URL: ${{ secrets.ARBITRUM_GOERLI_HOSTNAME_HTTP }}
+          ACCOUNTS_PRIVATE_KEYS: ${{ secrets.ARBITRUM_GOERLI_CONTRACT_OWNER_PRIVATE_KEY }}
+          ARBISCAN_GOERLI_API_KEY: ${{ secrets.ARBISCAN_GOERLI_API_KEY }}
+        run: yarn deploy --network goerliArbitrum
+
+      - name: Bump up package version
+        id: npm-version-bump
+        uses: keep-network/npm-version-bump@v2
+        with:
+          work-dir: cross-chain/arbitrum
+          environment: goerli-arbitrum
+          branch: ${{ github.ref }}
+          commit: ${{ github.sha }}
+
+      - name: Publish to npm
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm publish --access=public --tag goerli-arbitrum --network=goerliArbitrum
+
+      - name: Notify CI about completion of the workflow
+        uses: keep-network/ci/actions/notify-workflow-completed@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          module: "github.com/keep-network/cross-chain/arbitrum"
```

Comment:
@dimpar , I just learned from @nkuba that the first three chunks of the module name need to represent the repository name (probably new that before, but already forgot). So we have to start with `github.com/keep-network/tbtc-v2`. I propose either `github.com/keep-network/tbtc-v2/arbitrum` or `github.com/keep-network/tbtc-v2/cross-chain-arbitrum`.
Unfortunately, that means we need to update `config.json` in the `ci` repo again :(

Reply:
I've created https://github.com/keep-network/ci/issues/40 to not loose track of the work needed in `ci`.

Reply:
@michalinacienciala I've opened a new PR https://github.com/keep-network/ci/pull/41